src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -57,6 +57,55 @@ const RIPPLE_ANIMATION_CONFIG: RippleAnimationConfig = {
   exitDuration: numbers.FG_DEACTIVATION_MS,
 };
 
+/** Check box adapter. */
```

Comment:
This comment is not useful, you can just change it to `/** @docs-private */` so that this class is hidden from our docs